docs/README.md

```diff
@@ -100,46 +114,70 @@ or on Linux with
 xdg-open docs/target/html/index.html
 ```
 
-Or you can directly call your browser of choice with the filename e.g on Ubuntu
+Or you can directly call your browser of choice with the same filename. For example, on Ubuntu
 with Chromium:
 
 ```bash
 chromium-browser docs/target/html/index.html
 ```
 
-Alternatively, you can start a web server with that folder as root, e.g. again
-with Python and then open [http://localhost:4000](http://localhost:4000) in a
-web browser.
+Alternatively, you can start a web server with that folder as root, such as
+with the following Python command. You can then open [http://localhost:4000](http://localhost:4000)
+in a web browser.
 
 ```bash
 cd docs/target/html/
 python3 -m http.server 4000
 ```
 
 In order to see any changes from the source files in the HTML output, simply
-re-run the make command and refresh the browser.
+re-run the ``build`` command and refresh the browser.
 
 ## Versioning
 
-The version displayed in the resulting HTML is read from the top level Maven
-`pom.xml` file `version` field, by default.
+The version displayed in the resulting HTML is read by default from the top level Maven
+`pom.xml` file `version` field.
 
-To deploy a specific documentation set (e.g. a SNAPSHOT version) as release
-version you have to override the pom version with the `TRINO_VERSION`
+To deploy a specific documentation set (such as a SNAPSHOT version) as the release
+version you must override the pom version with the `TRINO_VERSION`
 environment variable.
 
 ```bash
-TRINO_VERSION=327 docs/build
+TRINO_VERSION=355 docs/build
 ```
 
 If you work on the docs for more than one invocation, you can export the
-variable and use it with sphinx.
+variable and use it with Sphinx.
 
 ```bash
-export TRINO_VERSION=327
+export TRINO_VERSION=354
 docs/build
 ```
 
 This is especially useful when deploying doc patches for a release where the
 Maven pom has already moved to the next SNAPSHOT version.
 
+## Contribution requirements
+
+
+To contribute corrections or new explanations to the Trino documentation requires
+only a willingness to help and submission of your [Contributor License
+Agreement](https://github.com/trinodb/cla).
+
+## Workflow
+
+The procedure to add a documentation contribution is the same as for [a code
+contribution](https://trino.io/development/process.html).
+
+* Discuss a small change in the #docs channel of the [Trino
+  Slack](https://trino.io/slack.html).
+* For a larger contribution, create a GitHub pull request as described
+  in [GitHub
+  documentation](https://docs.github.com/en/github/getting-started-with-github).
+  In brief, this means:
+  * [Create a fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) of the
+    [trinodb/trino](https://github.com/trinodb/trino) repository.
+  * Create a working branch in your fork.
+  * Make your edits in your working branch.
+  * In a browser, open your fork in GitHub, which offers to submit a pull
```

Comment:
Can users add the "Docs" label?

Reply:
With CLA in place, I just successfully added ``docs`` to [7683](https://github.com/trinodb/trino/issues/7683). So I assume that CLA is all that is required. 

According I added ``docs`` label to the Workflow.